montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -33,8 +36,27 @@ def load(self, solver, **kwargs):
         tf = self.tables['freq']
 
         na, nbl, ntime, nchan = solver.na, solver.nbl, solver.ntime, solver.nchan
-
-        # Check that we're getting the correct shape...
+	data_order = kwargs.get('data_order', ORDER_TIME_BL)
+        
+	# Define transpose axes to convert file uvw order 
+	# to montblanc array shape: (ntime, nbl)
+        if data_order == ORDER_BL_TIME:
+                file_uvw_shape = (nbl, ntime, 3)
+                uvw_transpose = (2,1,0)
+		file_ant_shape = (nbl, ntime)
+		ant_transpose = (1,0)
+		file_data_shape = (nbl, ntime, nchan,4)
+                data_transpose = (3,1,0,2)
+        elif data_order == ORDER_TIME_BL:
+                file_uvw_shape = (ntime, nbl,  3)
+                uvw_transpose = (2,0,1)
+		file_ant_shape = (ntime, nbl)
+		file_data_shape = (ntime, nbl, nchan,4)
+                data_transpose = (3,0,1,2)
+	else:
+		raise ValueError('Invalid UVW ordering %s', uvw_order)
+        
```

Comment:
@marziarivi There are tabs in the file and the spacing has gone a bit out. Each indent should be 4 spaces. No tabs should be present in the file.

Please check the indentation agrees with the general indentation of the file before these changes were made.